docs/connector-development/tutorials/building-a-python-source.md

```diff
@@ -19,16 +19,16 @@ All the commands below assume that `python` points to a version of python &gt;3.
 ### Creating a Source
 
 * Step 1: Create the source using template
-* Step 2: Build the newly generated source 
-* Step 3: Set up your Airbyte development environment 
+* Step 2: Build the newly generated source
+* Step 3: Set up your Airbyte development environment
 * Step 4: Implement `spec` \(and define the specification for the source `airbyte-integrations/connectors/source-<source-name>/spec.yaml`\)
 * Step 5: Implement `check`
 * Step 6: Implement `discover`
 * Step 7: Implement `read`
 * Step 8: Set up Standard Tests
 * Step 9: Write unit tests or integration tests
 * Step 10: Update the `README.md` \(If API credentials are required to run the integration, please document how they can be obtained or link to a how-to guide.\)
-* Step 11: Add the connector to the API/UI \(by adding an entry in `airbyte-config-oss/init-oss/src/main/resources/seed/source_definitions.yaml`\)
+* Step 11: Add the connector to the API/UI \(by adding a `metadata.yaml` file to the connectors folder\)
```

Comment:
It should already exists in  a new connector project because it's added by the generator.
```suggestion
* Step 11: Update the `metadata.yaml` file with accurate information about your connector. These metadata will be used to add the connector to Airbyte's connector registry.
```